src/components/Modals/ClaimNameFatFingerModal/ClaimNameFatFingerModal.tsx

```diff
@@ -53,7 +53,7 @@ export default class ClaimNameFatFingerModal extends React.PureComponent<Props,
             <Button secondary onClick={this.handleClose} type="button">
               {t('global.cancel')}
             </Button>
-            <Button primary type="submit" disabled={areNamesDifferent} loading={isLoading}>
+            <Button primary type="submit" disabled={areNamesDifferent || isLoading} loading={isLoading}>
```

Comment:
Maybe the button should be implicitly disabled when the loading prop is true

Reply:
That is not a bad idea at all, but we're using this button (that comes from semantic-react-ui) everywhere and we are going to be introducing a breaking change.